packages/orchestration/README.md

```diff
@@ -17,16 +17,145 @@ $ npm install @sap-ai-sdk/orchestration
   - Create a `.env` file in the sample-code directory.
   - Add an entry `AICORE_SERVICE_KEY='<content-of-service-key>'`.
 
-## Usage
+## Orchestration Client
 
-<!-- Details to be added -->
+The orchestration client provides essential features like templating and content filtering, which are often required in business AI scenarios:
 
-## Support, Feedback, Contribution
+- **Templating** allows composing prompts with placeholders that can be filled during a chat completion request.
+- **Content filtering** lets you restrict the type of content sent to or received from a generative AI model.
```

Comment:
[pp] "type of content" is not really the key here

Reply:
Copied it directly from the [official docs](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration) though ;)